src/webots/nodes/WbRobot.cpp

```diff
@@ -727,6 +734,22 @@ void WbRobot::keyReleased(const QString &text, int key) {
   emit keyboardChanged();
 }
 
+void WbRobot::writeDeviceConfigure(QList<WbDevice *> devices, QDataStream &stream) const {
+  QListIterator<WbDevice *> it(devices);
+  while (it.hasNext()) {
+    WbDevice *device = it.next();
```

Comment:
```suggestion
    const WbDevice *device = it.next();
```

(I know this was already like this)